.github/actions/setup-bubblewrap/action.yaml

```diff
@@ -0,0 +1,55 @@
+# Copyright 2025 Chainguard, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+name: 'Setup Bubblewrap'
+description: 'Make bubblewrap work on ubuntu-latest'
+# See https://github.com/chainguard-dev/melange/issues/1508
+
+inputs:
+  path:
+    description: 'Path to the program that needs to run bubblewrap'
+    required: true
+    default: '/usr/bin/melange'
+
+runs:
+  using: "composite"
+  steps:
+    - name: Disable apparmor userns restrictions
+      shell: bash
+      run: |
+        sudo bash -c "mkdir -p /etc/sysctl.d
+        echo 'kernel.apparmor_restrict_unprivileged_userns = 0' >> /etc/sysctl.d/60-apparmor-namespace.conf"
+
+    - name: Allow bubblewrap to use unprivileged user namespaces independent of who calls it
+      shell: bash
+      run: |
+        sudo bash -c "cat << EOF > /etc/apparmor.d/local-bwrap
+          abi <abi/4.0>,
+          include <tunables/global>
+
+          profile local-bwrap /usr/bin/bwrap flags=(unconfined) {
+            userns,
+
+            # Site-specific additions and overrides. See local/README for details.
+            include if exists <local/bwrap>
+          }
+        EOF
+
+        systemctl reload apparmor"
+
+    - name: Allow melange to use unprivileged user namespaces
```

Comment:
pretty sure you do not need this hunk if you have the one above (you let anyone call bwrap, and melange is included in that anyone).

Reply:
Good call, let's find out: https://github.com/chainguard-dev/melange/pull/1882